Src/Base/AMReX_MultiFabUtil_2D_C.H

```diff
@@ -259,7 +260,7 @@ void amrex_avgdown (Box const& bx, Array4<Real> const& crse,
         for (int i = clo.x; i <= chi.x; ++i) {
             int ii = i*facx;
             int jj = j*facy;
-            Real c = 0.;
+            T c = {};
```

Comment:
@WeiqunZhang 
Is there a better way to do template-safe zero initialization? There are many libraries (like `eigen`) that do not use default constructors as zero constructors.

Reply:
`auto c = T(0);`?

Reply:
Or `T c = 0;`